HERESUP-27616 IAM-6079 Replace ini4j dependency to apache commons-con…

ashishKhushiKumar opened the pull request:
…figuration2

ashishKhushiKumar commented:
Addressing the vulnerabilities mentioned https://github.com/heremaps/here-aaa-java-sdk/security/dependabot in the pull request.

spaltis reviewed (commented):
Can you pls store the updated library in local maven repo and import it into authz sdk and run tests locally?

ashishKhushiKumar commented on pom.xml:
```diff
@@ -65,11 +65,11 @@
         <junit.version>4.13.1</junit.version>
         <mockito.version>1.10.19</mockito.version>
-        <ning.version>1.8.17</ning.version>
+        <ning.version>2.12.4</ning.version>
```
Ack.

ashishKhushiKumar commented:
CC: @spaltis @owenkellett I have raised the final revision with the following changes - 

1. Replaced org.ini4j with org.apache.commons.commons-configuration2 to handle INI files parsing.
2. Replaced com.ning with org.asynchttpclient with version 2.12.4 which is the latest in 2.x series.
3. Upgraded jackson-databind version from 2.13.3. to 2.19.1.
